index.html

```diff
@@ -12,10 +12,97 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<!--navigation bar -->
+<header id="header">
+    <div class="logo"><img alt="karma" id="header-img" src="http://thinkful-ed.github.io/karma-clone/img/karma-logo.svg"></div>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <nav id="nav-bar">
+       <ul>
+        <li>
+            <a class="nav-link" href="#meet-karma">Meet Karma</a>
+        </li>
+        <li>
+            <a class="nav-link" href="#how-it-works">How it Works</a>
+        </li>
+        <li>
+            <a class="nav-link" href="#store">Store</a>
+        </li>
+        <li>
+        <a class="nav-link" href="#Blog">Blog</a>
+         </li>
+          <li>
+        <a class="nav-link" href="#Help">Help</a>
+          </li>
+          <li>
+        <a class="nav-link" href="#Long">Login</a>
+          </li>
+           </ul>
+     </nav>
+</header>
+<!--navigation End -->
+
+<section class="showcase">
+    <h2>
+        Introducing Karma
+    </h2>
+    <h4>
+        Bring WiFi with you, everywhere you go.
+    </h4>
+
+    <button class="button" type="button">
+        Learn More
+    </button>
+
+</section>
+
+ <!-- SEGMENTS -->
+<section class="seg">
+
+ <h2>
+    Everyone needs a little Karma.
+ </h2>
+ <div class="segments">
+    <div class="devices">
+        <img src="http://thinkful-ed.github.io/karma-clone/img/icon-devices.svg" alt="Devices" class="dev ">
```

Comment:
Rather than using a full URL for this image it's better to use a relative one 

```
../img/icon-devices.svg
````